plasmapy/formulary/tests/test_mathematics.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the mathematics.py module."""
+
+
+import numpy as np
+import pytest
+
+from plasmapy.formulary import mathematics as mathematics
+
+# TODO: Move the Fermi integral tests over to this file?
+
+
+def test_rot_a_to_b():
+
+    # Test one rotation
+    a = np.array([1, 0, 0])
+    b = np.array([0, 1, 0])
+    R = mathematics.rot_a_to_b(a, b)
+    correct = np.array([[0, 1, 0], [-1, 0, 0], [0, 0, 1]])
+    assert np.allclose(R, correct, atol=1e-6)
```

Comment:
```suggestion
    np.testing.assert_allclose(R, correct, atol=1e-6)
```
https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html has a bit nicer output when something breaks, so i like it for array comparisons